packages/testing/src/testing.ts

```diff
@@ -89,7 +90,7 @@ export function createTestingPinia({
 
   pinia.use(({ store }) => {
     if (initialState[store.$id]) {
-      store.$patch(initialState[store.$id])
+      mergeReactiveObjects(store.$state[store.$id], initialState[store.$id])
```

Comment:
didn't you mean

```suggestion
      mergeReactiveObjects(store.$state, initialState[store.$id])
```

Reply:
I don't think so - wouldn't this be

```
const $state = {
   module: {
     isBoolean: true
   }
}

const initialState = {
  module: {
     isBoolean: false
   }
}

mergeReactiveObjects($state['module'], initialState['module'])

/* $state = {
  module: {
     isBoolean: false
   }
} */
```

Where as the suggestion looks like it would do:

```ts
mergeReactiveObjects($state, initialState['module'])

/* $state = {
  module: {
     isBoolean: true
   },
   isBoolean: false
} */
```

Reply:
Ah wait so yes - forgot each store is not nested like Vuex :D

Reply:
initialState is meant to be the rootState while `store.$state` is a store's local state